### DISCUSSION TITLE
Tuning parameters inoperative

### DISCUSSION THREAD

**Discussion by Jean-Gabriel1982** [GrandOrgue/grandorgue#1423] [General]
hello, in the parameters of the organ in use, today on my brother's computer (linux mint) with the latest version of GO, the parameters Amplitude, Gain, Tuning ... are inoperative.

whatever the parameter chosen, after recording, no change is visible.

for example, I took the Contrebasse 16' (GO demo organ), modified the Gain to +10, saved, then modified the Tuning to -1200 (to get the result of 32'), saved.

after committing the changes, nothing happens. no increase in sound, no modification of the octave...

**Comment by ClarkeBrunt** [comment]
This is because you happen to have chosen a rank in the GO demo organ which shares the pipes of another rank (that's why the demo organ is so small - many ranks share pipes).

For example, again the demo organ, in the Organ settings dialog I chose Montre 8', changed Manual Tuning, did Apply, and the pitch changed (after re-playing a note - if the note is still playing, it doesn't change).

Playing a note using Prestant 4', the pitch had also changed. This is because this rank 'borrows' the pipes from Montre 8' but an octave higher. Any changes to pitch etc. for Prestant 4' have no effect. Contrebasse 16' also shares the same pipes as Montre 8'

If suppose this might seem illogical. You can see in the Organ Settings dialog that Prestant 4' has no pipes within it. I guess you are to imagine a real physical organ. If you tune one of the physical pipes, then it will affect any ranks which happen to share that pipe.

You can see the sharing in the demo.organ file (if you uncompress the 'orgue' file) - 7-zip, gzip, etc. Lots of REF lines for shared pipes.

**Comment by oleg68** [comment]
Thank you, @ClarkeBrunt.

@Jean-Gabriel1982 These parameters only affect ranks containing own pipes. If the rank has only borrowed pipes, it is not affected.

Unfortunally, the demo organ has lot of borrowing. Use another sample set for fine-grain tuning.

**Reply by Jean-Gabriel1982** [reply]
Thank you @ClarkeBrunt and @oleg68 for your answers ... but before I've installed the last GO version, my brother had the 0.3 ... and it worked with the Contrebasse 16'. I put -1200 and had the 32' 🫣

**Comment by oleg68** [comment]
Contrebasse 16' borrows the pipes with Montre 8'. So you can only change their pitches together. If you need 32' , put -1200 to Montre 8'.
